Automattic/newspack-plugin
# Remove "Back to dashboard" button

### All Submissions:






### Changes proposed in this Pull Request:

There are two ways to access a Wizard:

1. via the dashboard
2. via the sidebar menu

Having a button telling you to go "back to dashboard" doesn't make sense if you went via (2.) so it's probably better to completely remove it.

There was also an inconsistency between Wizards. Some of them had the button, some of them didn't.

_Note: users are still able to go "back to dashboard" by clicking on the Newspack Logo or via the sidebar_

### How to test the changes in this Pull Request:

1. Switch to this branch
2. Test every Wizards and check for no "Back to dashboard" buttons

### Other information:
